src/documents/articles/a-quick-polymer-introduction.html.md

```diff
@@ -57,14 +58,28 @@ There are a few wrappers around web components such as [x-tag](http://x-tag.read
 
 The aim of the Polymer project is not to create a bloated library, but instead make the smallest library possible, and make the browser do all the work.
 
-\#UseThePlatform is the name Polymer give to represent the want for the browser to do all the heavy lifting without libraries.
+[\#UseThePlatform](https://www.polymer-project.org/1.0/blog/2016-05-26-IO-2016-Recap.html) is the name Polymer give to represent the want for the browser to do all the heavy lifting without libraries.
 
-One of the advantages of using Polymer is that it has a built in data binding model (like [React](https://facebook.github.io/react/) or [Angular](http://angularjs.org/)) enabling complete applications to be made without any other libraries.
+One of the advantages of using Polymer is that it has a built in data binding model (like [Angular](https://angular.io)) enabling complete applications to be made without any other libraries.
 
-Polymer [supports evergreen browsers](https://www.polymer-project.org/1.0/docs/browsers) and when combined with webcomponents.js supports IE 11+.
+Polymer [supports all major evergreen browsers](https://www.polymer-project.org/1.0/docs/browsers) and when combined with webcomponents.js supports IE 11+.
```

Comment:
`all major` to `all the major`. Isn't IE11 evergreen as well? Maybe not but I thought it auto updated